app/views/setup/discovery.html.slim

```diff
@@ -3,6 +3,13 @@
   span
     | A supported deployment of SUSE CaaS Platform requires a minimum of three nodes. Please select a minimum of three nodes.
 
+.alert.alert-danger.discovery-bootstrap-alert role="alert" hidden="true"
+  i.fa.fa-4x.pull-left aria-hidden="true"
+  span
+    p This cluster is not bootstrappable. The reasons are:
```

Comment:
i think `bootstrappable` as an internal name is fine, but in the UI i'd call it differently, maybe:

```
Unable to bootstrap cluster:

* list
* list
```

Reply:
Done!